readme.txt

```diff
@@ -330,6 +330,7 @@ Please see the changelog for the complete list of changes in this release. Remem
 = [4.5.12.1] 2017-09-07 =
 
 * Fix - Fixed an issue where events imported via Event Aggregator from an iCal-like source would be duplicated in place of being updated [87654]
+* Fix - Fixed an issue where manually running Scheduled Imports would always result in a failed import [87321]
```

Comment:
Changed base to our hotfix branch (`release/M17.17.2`) ... let's also bump this changelog entry a little higher (into the `4.5.12.2` section).